src/Conversion/ONNXToKrnl/Math/Gemm.cpp

```diff
@@ -97,19 +97,19 @@ struct ONNXGemmOpLowering : public ConversionPattern {
     // Scope for krnl EDSC ops
     using namespace mlir::edsc;
     using namespace mlir::edsc::intrinsics;
-    // ScopedContext scope(rewriter, loc);
 
     // R is result (alloc).
     Value A(operandAdaptor.A()), B(operandAdaptor.B()), R(alloc);
-    MemRefBoundsCapture aBound(A), rBound(R);
+    MemRefBoundsIndexCapture aBound(A), rBound(R);
```

Comment:
Use IndexExpr for iterates... it make it use constant, and that prevent an annoying warning from affine.